deployment/united-manufacturing-hub/values.yaml

```diff
@@ -338,6 +338,20 @@ _001_customMicroservices:
 serviceAccount:
   create: true
 
+### automigrate ###
+automigrate:
+  enabled: true
+  image:
+    repository: unitedmanufacturinghub/automigrate
+    pullPolicy: IfNotPresent
+  resources:
+    requests:
+      memory: 2Gi
+      cpu: 1000m
```

Comment:
This is very high and might result in the system being unable to start on amaller development setups

Reply:
Reduced